Processes -- Lauren Dubose

Lnleite opened the pull request:
- [x] ex1
- [x] ex2
- [x] ex3
- [x] ex4
- [x] ex5
- [x] ex6
- [x] Stretch

Lnleite committed:
First exercise done

Lnleite committed:
Second exercise done

Lnleite committed:
Manipulate code for second exercise

Lnleite committed:
Third Exercise done

RachelDiCesare93 commented on ex2/ex2.c:
```diff
@@ -1,14 +1,52 @@
+        fprintf(fp, "Look it is %s", txt);
+        printf("Parent descriptor: %d\n", fileNum);
+    }
+
```
Great job! Keep it up with the comments too, this will benefit you and your future employers will appreciate it!

RachelDiCesare93 commented on ex1/ex1.c:
```diff
@@ -2,13 +2,36 @@
+        printf("Parent x:%d \n", x);
+    }
     return 0;
 }
```
Great job Lauren!

Lnleite committed:
Exercise 4 working

Going to play with more exec functions before moving on.

Lnleite committed:
Use execv

Lnleite committed:
Exercise five done

Learned that you have to call the pipe before the fork.

Lnleite committed:
Exercise six done

Lnleite committed:
Random amount function made

Lnleite committed:
Bank is operational

Lnleite committed:
Random number fix

The random number was not random because there was 2 srand in the file. It is
now fixed.

Lnleite committed:
Add exit(2) and answer most of the questions

Lnleite committed:
Add some of lecture examples

Lnleite committed:
Add prefered way to close pipes

Lnleite committed:
Add lecture code
